apps/docs/content/docs/sync.mdx

```diff
@@ -15,74 +15,36 @@ keywords:
   - websockets
 ---
 
-You can add realtime multi-user collaboration to your tldraw app by using **tldraw sync**. It's our library for fast, fault-tolerant shared document syncing, and is used in production on our flagship app tldraw.com.
+You can add realtime multi-user collaboration to your tldraw app by using **tldraw sync**. It's our library for fast, fault-tolerant shared document syncing, and is used in production on our flagship app [tldraw.com](https://tldraw.com).
 
-<div className="article__image" style={{ border: 'none' }}>
-	<img
-		alt="Multiplayer example using tldraw sync"
-		src="/images/multiplayer.gif"
-		style={{
-			// learn this one weird trick for transparent animated gifs. doctors hate her!!!
-			mask: 'url(/images/multiplayer_mask.png) luminance center 100% / 100% no-repeat',
-			WebkitMask: 'url(/images/multiplayer_mask.png) luminance center 100% / 100% no-repeat',
-		}}
-	/>
-</div>
+While we offer a [hosted demo](/collaboration#tldraw-sync-demo) of the tldraw sync library for prototyping, the demo is suitable only for prototyping. To use tldraw sync in production, you will need to self-host the tldraw sync server.
```

Comment:
this feels clunky - double "for prototyping" in 1 sentence, double "tldraw sync" in the next.